app/controlplane/internal/authz/authz.go

```diff
@@ -181,8 +180,9 @@ var ServerOperationsMap = map[string][]*Policy{
 	// Robot Account
 	"/controlplane.v1.RobotAccountService/List": {PolicyRobotAccountList},
 	// Workflows
-	"/controlplane.v1.WorkflowService/List": {PolicyWorkflowList},
-	"/controlplane.v1.WorkflowService/View": {PolicyWorkflowRead},
+	"/controlplane.v1.WorkflowService/List":   {PolicyWorkflowList},
```

Comment:
we need to revisit the read only roles up there to make sure they still work.

Note that authz and the middleware is used for both apitokens but also users via roles. We should probably map the read only policies you put there to the reader role if we have not done it yet.

Reply:
Yes, it seems those policies are already added to the reader role.